CMSgov/design-system
# [WNMGDS-3007] Fix arrow behavior for disabled tabs

## Summary
Updates the arrow key navigation logic in `Tabs` components so that the content of disabled tabs is no longer reachable or visible using arrow keys.

[Jira Ticket](https://jira.cms.gov/browse/WNMGDS-3007) 

## How to test

- Run `yarn storybook`
- Inspect disabled stories for `Tabs` and/or `ds-tabs`.
- Use `tab` key to initialize focus, then use left and right arrow keys for navigation.
- Repeat the above steps while using a screen reader to confirm that disabled tabs are skipped and their content is not announced.

## Checklist
